### PR TITLE
Use new directory structure in ViViaN

### PR DESCRIPTION
**Pull request by BetsyMcPhail**
-- top-level directory
 ---- files
 ---- vivian
 ---- dox

### CODE REVIEW

File: Visual/index.php [hunk 1]
```diff
@@ -85,7 +85,7 @@
 
 <?php include_once "vivian_tree_layout_common.js" ?>
 
-var package_link_url = "http://code.osehra.org/dox/";
```
Comment (BetsyMcPhail) [inline Visual/index.php]: @josephsnyder - I'm not sure what's correct for this link... Related issue is here https://issues.osehra.org/browse/VIVIAN-515

**Comment by BetsyMcPhail** [comment]
@josephsnyder the files dir argument has been added to the setup script, ready for another look

**Comment by BetsyMcPhail** [comment]
@josephsnyder I added another commit to address https://issues.osehra.org/browse/VIVIAN-515.
Let me know what you think!

**Comment by BetsyMcPhail** [comment]
@josephsnyder ready for another review with filtering re-introduced

**Comment by BetsyMcPhail** [comment]
Includes changes for direct link from ViViaN to DOX (https://issues.osehra.org/browse/VIVIAN-516)

**Comment by josephsnyder** [comment]
Approval still stands after all of the recent changes.  Ready to merge when the DOX pages for November are uploaded

**Pull request merged**